src/components/chips/chips.spec.js

```diff
@@ -659,6 +659,59 @@ describe('<md-chips>', function() {
 
       });
 
+      describe('required', function() {
+        beforeEach(function() {
+          // Clear default items to test the required chips functionality
+          scope.items = [];
+        });
+
+        it('should set the required error when chips is compiled with an empty array', function() {
+          var template =
+              '<form name="form">' +
+              '<md-chips name="chips" required ng-model="items"></md-chips>' +
+              '</form>';
+
+          var element = buildChips(template);
+          element.scope().$apply();
+
+          expect(scope.form.chips.$error['required']).toBe(true);
```

Comment:
Let me know if you want me to switch to the dot notation, I was just copying how the md-max-chips validator test did it, but realize now that it used bracket notation since has special characters in the name.